pkg/controllers/leaderworkerset_controller.go

```diff
@@ -389,6 +390,9 @@ func (r *LeaderWorkerSetReconciler) updateConditions(ctx context.Context, lws *l
 	updateCondition := setCondition(lws, condition)
 	// if condition changed, record events
 	if updateCondition {
+		if updatedCount == int(*lws.Spec.Replicas) {
+			metrics.ReplicaReadyStatus(readyLeaderPods, getLastTransitionTime(string(leaderworkerset.LeaderWorkerSetProgressing), lws))
```

Comment:
the start time here is when the lws just progress, there might be a possibility that the leader pod is not scheduled.

Reply:
Asked a similar question in other comment: what is the replica latency actually measuring? I thought it was from lws creation to when the replica is ready